apps/agreement/contracts/example/RegistryApp.sol

```diff
@@ -0,0 +1,289 @@
+/*
```

Comment:
A bit of clean up, but perhaps we should make this example a part of aragonOS instead?

I think we may also have to update this with the new interfaces too (so a good thing to do as a second PR).